proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java

```diff
@@ -315,6 +316,23 @@ public void onNetworkError(Throwable t) {
         }
     }
 
+    /**
+     * Returns reference to {@code PowerAuthTokenStore} instance. The internal instance is created on demand, when
+     * the getter is called for first time.
+     *
+     * @return Reference to {@code PowerAuthTokenStore} instance.
+     */
+    public PowerAuthTokenStore getTokenStore() {
```

Comment:
Not sure if this is correct, see https://en.wikipedia.org/wiki/Double-checked_locking#Usage_in_Java ("// Broken multithreaded version")...

Reply:
My mistake, I've fixed that and the method is fully synchronized.